config/routes.rb

```diff
@@ -7,6 +7,7 @@
   get 'courses/:id/terms/:term_number', to: 'courses#show', as: "show_term"
   put 'courses/:id/terms/new', to: 'courses#add_term', as: "add_term"
   get 'courses/:id/statistics', to: 'courses#statistics', as: "show_statistics"
+  get 'courses/:id/terms/:term_number/settings', to: 'courses#settings', as: "settings_term"
```

Comment:
Do we really need a separate page for course settings? What kind of settings will be there?